modules/microshift-nw-advertise-address.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-yaml-advertiseAddress_{context}"]
+= Configuring the advertise address network flag
+
+The `apiserver.advertiseAddress` flag specifies the IP address on which to advertise the API server to members of the cluster. This address must be reachable by the cluster. You can set a custom IP address here, but you must also add the IP address to a host interface. Customizing this parameter preempts {microshift-short} from adding a default IP address to the `br-ex` network interface.
+
+[IMPORTANT]
+====
+If you customize the `advertiseAddress` IP address, make sure it is reachable by the cluster when {microshift-short} starts by adding the IP address to a host interface.
+====
+
+If unset, the default value is `10.44.0.0/32`. It will be set to the next immediate subnet after the service network. For example, when the service network is `10.43.0.0/16`, the `advertiseAddress` is set to `10.44.0.0/32`.
```

Comment:
I would remove the first part about the default value being '10.44.0.0/32', as it is explained right after that. There is also another example too.